stock-tracker/package.json

```diff
@@ -41,5 +41,10 @@
   },
   "devDependencies": {
     "nodemon": "^1.19.2"
+  },
+  "husky": {
```

Comment:
You are using a number of dependencies you haven't added to your project's `devDependencies`:
1. `husky`
2. `prettier`
3. `pretty-quick`

You need to add those, else it's very difficult for any other developer working on your project.